src/validation/IntegerString.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import {Validation, ValidationTree} from "./Validation";
+import {native} from "./Native";
+
+export const integerString: Validation = {
+    type: 'integerString',
+    validate(value: any, key?: string, validation?: ValidationTree): ValidationTree {
+        const node = key !== undefined && validation !== undefined ? native('string').validate(value, key, validation) : native('string').validate(value);
+        if (typeof value === "string") {
+            if (value.search(/^-?[0-9]+$/) === -1) {
```

Comment:
Shouldn't we also accept a leading `+` sign?

Reply:
[Done.](https://github.com/digital-asset/daml-js/pull/59/commits/3e7c2925ecf0b93eb36819ff07ab2b9c03199f26)